stanovy_vpsFree.cz.md

```diff
@@ -43,6 +43,18 @@ standardů.
 5. poskytování poradenských a konzultačních služeb,
 6. spolupráce se subjekty, které sledují podobné cíle.
 
+### §3a Vedlejší činnost
+
+1. Spolek může vykonávat vedlejší činnost.
+2. Vedlejší činností je hospodářská činnost vykonávaná za účelem podpory
+hlavních činností Spolku a za účelem hospodárného využití majetku Spolku. K
+zajištění svých hlavních činností může Spolek vykonávat podnikatelskou činnost
+na základě oprávnění podle zákona č. 455/1991 Sb., o živnostenském podnikání,
+ve znění pozdějších předpisů, v rámci živnosti volné „Výroba, obchod a služby
+neuvedené v přílohách 1 až 3 živnostenského zákona.
+3. Veškerý zisk získaný Spolkem z vedlejší činnosti musí být použit v plném
+rozsahu na hlavní činnost Spolku.
```

Comment:
ianal ale znamena tohle, ze zisk z vedlejsi cinnosti nelze pouzit na jinou vedlejsi cinnost? Je to umyslne omezeni?

Reply:
ano, to je schválně